lib/jobi/job.rb

```diff
@@ -6,12 +6,13 @@ class Job
     class << self
       include Utils
 
-      def options(queue_name:, ack: false, consumers: 5, durable: true, persist: false)
-        @queue_name = queue_name.to_s
-        @ack = ack
-        @consumers = consumers
-        @durable = durable
-        @persist = persist
+      def options(options = {})
+        @queue_name = options[:queue_name].to_s
+        @qos = options[:qos] || 5
```

Comment:
I have checked Kafka's implementation of the prefetching, they mentioned nothing about QoS, I suggest we rename it to prefetch to make things clearer for the users

```suggestion
        @qos = options[:prefetch] || 5
```

Reply:
Okay